src/LazyOperations/LinearMap.jl

```diff
@@ -96,13 +96,13 @@ EmptySet{Int64}(3)
 ```
 """
 struct LinearMap{N,S<:LazySet{N},NM,
-                 MAT<:AbstractMatrix{NM}} <: AbstractAffineMap{N,S}
+                 MAT<:Union{AbstractMatrix{NM},MatrixZonotope{NM}}} <: AbstractAffineMap{N,S}
```

Comment:
Is it intentional that `LinearMap` cannot receive a general `AbstractMatrixZonotope`?